MovingBlocks/TerasologyLauncher
# fix: launcher logo

### How to test

1. Start the launcher without the changes of this PR
2. Check that you cannot see a logo in the upper left corner of the launcher UI
3. Start the launcher with the changes of this PR
4. Verify that you can see the logo containing gooey and the text "terasology" in the upper left corner
